specs/protocol/README.md

```diff
@@ -362,37 +362,34 @@ The `metadata` object contains fields _about_ the message and is present in _eve
 The actual message content. This will _always_ be a JSON object. The [Message Kinds section](#message-kinds) specifies the content for each individual message type
 
 
-### `private`
-Often times, an RFQ will contain PII or PCI data either within the `credentials` being presented or within `paymentDetails` of `payinMethod` or `payoutMethod` (e.g. card details, phone numbers, full names etc). 
+### `privateData`
+Often times, an RFQ will contain PII or PCI data either within the `claims` being presented or within `paymentDetails` of `payin` or `payout` (e.g. card details, phone numbers, full names etc).
 
-In order to prevent storing this sensitive data with the message itself, the value of a property containing sensitive data can be a [hash](#Hashing) of the sensitive data. The actual sensitive data itself is included in the `private` field. 
+In order to prevent storing this sensitive data with the message itself, an RFQ may have property `privateData` which holds the raw values for certain fields, while the RFQ's `data` property holds salted hashes of the fields in `privateData`. `privateData` is NOT signed over, so the signature integrity of an RFQ can be verified even if `privateData` is not present. The hashed value is produced by creating a [digest](#digests) of a JSON array containing a salt and the cleartext value that appears in `privateData`. For example, to produce `Rfq.data.payin.paymentDetailsHash`, create a digest of `[salt, Rfq.privateData.payin.paymentDetails]`. The RECOMMENDED minimum length of the randomly-generated portion of the salt is 128 bits.
 
-The `private` field is ephemeral and **MUST** only be present when the message is initially sent to the intended recipient
+Message kinds other than RFQ may NOT have property `privateData`. The [RFQ `privateData` section](#rfq-request-for-quote) specifies the content of RFQ `data` and `privateData`.
 
-The value of `private` **MUST** be a JSON object that matches the structure of `data`. The properties present within `private` **MUST** only be the properties of `data` that include the hash counterpart.
-
-> **Note**
-> Rationale behind the `private` JSON object matching the structure of `data` is to simplify programmatic hash evaluation using JSONPath to pluck the respective hash from `data`. **NOTE**: we should try this to make sure it's actually "easy"
+The `privateData` field is ephemeral and **MUST** only be present when the message is initially sent to the intended recipient
```

Comment:
i think we need to explain the `*Hashes` properties in this section. it's currently only explained in an `e.g.`

Reply:
Added a more explicit section for `*Hashes`

Reply:
nice. reads really well! looks like there's a table formatting issue 

<img width="1216" alt="image" src="https://github.com/TBD54566975/tbdex/assets/4887440/e32b180d-518c-4611-aad1-f9de4fa6d4a9">